web_tooltip_field/__init__.py

```diff
@@ -0,0 +1,5 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import models
+from . import fields
+from . import tests
```

Comment:
This import is not needs.

If you keep it all tests module will be loaded on production, this can be undesirable in some circumstance (importing test libraries...)

```suggestion
```